docker-compose.yml

```diff
@@ -0,0 +1,82 @@
+version: "3"
+
+services:
+
+  database:
+    image: "postgres:11"
+    environment:
+      POSTGRES_USER: omero
+      POSTGRES_DB: omero
+      POSTGRES_PASSWORD: omero
+    networks:
+      - omero
+    volumes:
+      - "database:/var/lib/postgresql/data"
+
+  omeroserver:
+    build: ./server
+    environment:
+      CONFIG_omero_db_host: database
+      CONFIG_omero_db_user: omero
+      CONFIG_omero_db_pass: omero
+      CONFIG_omero_db_name: omero
+      CONFIG_omero_server_nodedescriptors: >-
+        master:Blitz-0
+        omeroworker-1:Processor-0,Tables-0
+        omeroworker-2:Indexer-0,PixelData-0
+        omeroworker-3:DropBox,MonitorServer,FileServer,Storm
+      CONFIG_omero_master_host: omeroserver
+      ROOTPASS: omero
+    networks:
+      - omero
+    ports:
+      - "4063:4063"
+      - "4064:4064"
+    volumes:
+      - "omero:/OMERO"
+
+  omeroworker-1:
```

Comment:
This might be doable with just the regular docker-compose `scale` functionality. I assume then there would be another value which would replace `OMERO_WORKER_NAME`. But that can all be handled in a follow-up.

Reply:
Unfortunately I don't think it's that easy, since the master node needs to know about all worker nodes. AFAICT `scale` only spins up more copies of a container, so we'd need a way to notify the master node (and ensure it's restarted/reloaded?)

Reply:
Good point.